pkg/cloudevents/event.go

```diff
@@ -27,6 +27,10 @@ func (e Event) Type() string {
 	return e.Context.GetType()
 }
 
+func (e Event) ExtensionAs(name string, obj interface{}) {
```

Comment:
Please add doc strings to this and the other public methods in this PR.

Reply:
Done! Thanks!